packages/circuit-ui/components/Button/__snapshots__/Button.spec.tsx.snap

```diff
@@ -1,570 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
There are use cases where a snapshot of the DOM can be useful – I'm thinking of things like the `scope` attributes we had on the Table component, that Chromatic couldn't catch because they're not visual but still important for accessibility.

Maybe for the Button they're not as important, but shouldn't we still keep the snapshots around for this kind of cases?

Reply:
I've reverted this change for now.